test/libyul/objectCompiler/subObjectAccess.yul

```diff
@@ -0,0 +1,79 @@
+object "A" {
+  code {
+    pop(dataoffset("A"))
+    pop(dataoffset("B"))
+    pop(dataoffset("B.C"))
```

Comment:
I'm a bit concerned that this pulls in `B.C` as a new sub-assembly. I think it would be better if we could either directly reach into the sub-sub-assembly or if we would have a requirement to perform the copy already in the yul source code.